lesson-18/index.js

```diff
@@ -0,0 +1,194 @@
+"use strict"
+
+
+// task1
+
+// function getSum(n) {
+//     let sum = 0;
+   
+//     for (let i = 0; i <= n; i += 1) {
+//        sum += i;
+//     }
+
+//     return sum;
+// }
+
+// let x = +prompt();
+// alert( getSum(x) );
+ 
+
+//task2
+
+// let percentageYear = 0.17;
+// let numberOfYears = 5;
+
+// function credit(n) {
+
+//     return ( n * (percentageYear * numberOfYears) );
+// }
+
+// let x = +prompt();
+// alert( credit(x) );
+
+
+//task3
+
+// function trimString(n) {
+//     let x = prompt("");
+//     let y = prompt("");
+
+//     return n = str.substring(x, y);
+// }
+
+// let str = prompt("");
+// console.log( trimString(str) );
+
+
+//task4
+
+// function getNumbersSum(num) {
+//     let figures = "" + num;
+//     let sum = 0;
+      
+//     for (let i = 0; i < figures.length; i +=1) {
+//         sum += +figures[i];
+//     }
+
+//     return sum;
+// }
+    
+// let result = prompt("");
+// console.log( getNumbersSum(result) );
+
+
+//task5
+
+// let a = +prompt("Введите число");
+// let b = +prompt("Введите число");
+
+// function getSum() {
+
+//     if (a === b) {
+//         return alert(a && b);
+//     } else { 
+//         let sum = 0;
+
+//         while(a <= b) {       
+//             sum += a;  
+//             a ++;
```

Comment:
Лишний пробел. Также лучше 
```suggestion
//             a += 1;
```